scripts/validate-marketplace.sh

```
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Validate Claude Code marketplace and plugin structure
+
+set -euo pipefail
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)